src/Authoring/TestDiagnostics/TestDiagnostics.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <Platforms>x64;x86</Platforms>
+    <LangVersion>preview</LangVersion>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <CsWinRTEnableLogging>true</CsWinRTEnableLogging>
+    <CsWinRTComponent>true</CsWinRTComponent>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <GeneratedFilesDir Condition="'$(GeneratedFilesDir)'==''">$([MSBuild]::NormalizeDirectory('$(MSBuildProjectDirectory)', '$(IntermediateOutputPath)', 'Generated Files'))</GeneratedFilesDir>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <CompilerVisibleProperty Include="AssemblyName" />
```

Comment:
These are also set by our cswinrt targets when CsWinRTComponent = true so shouldn't be needed anymore.

Reply:
this project got deprecated in latest changes